app/controllers/repositories_controller.rb

```diff
@@ -357,21 +357,23 @@ def export_repositories
     if repositories.present? && current_user.has_available_exports?
       current_user.increase_daily_exports_counter!
       RepositoriesExportJob.perform_later(repositories.pluck(:id), user_id: current_user.id, team_id: current_team.id)
+      log_activity(:export_inventories, inventories: repositories.pluck(:name).join(', '))
       render json: { message: t('zip_export.export_request_success') }
     else
       render json: { message: t('zip_export.export_error') }, status: :unprocessable_entity
     end
   end
 
   def export_repository_stock_items
-    row_ids = @repository.repository_rows.where(id: params[:row_ids]).pluck(:id)
-    if row_ids.any?
+    repository_rows = @repository.repository_rows.where(id: params[:row_ids])
```

Comment:
such approach will result in 3 separate SQL queries. Optimize it

Reply:
@okriuchykhin Any suggestions please for optimizing here,
I am attempting to use ids for exporting items and the names for creating activities